pop-sandbox/src/lib.rs

```diff
@@ -0,0 +1,139 @@
+use core::any::Any;
+
+use frame_metadata::RuntimeMetadataPrefixed;
+use frame_support::{
+	sp_runtime::{traits::Header, AccountId32},
+	traits::Hooks,
+};
+use frame_system::pallet_prelude::BlockNumberFor;
+use pop_runtime_devnet::{BuildStorage, Runtime};
+use sp_io::TestExternalities;
+use frame_support::sp_runtime::traits::One;
+use pop_runtime_devnet::Balance;
+
+/// Alias for the account ID type.
+pub type AccountIdFor<R> = <R as frame_system::Config>::AccountId;
+
+/// Default initial balance for the default account.
+pub const UNIT: Balance = 10_000_000_000;
+pub const INIT_AMOUNT: Balance = 100_000_000 * UNIT;
+pub const INIT_VALUE: Balance = 100 * UNIT;
+pub const ALICE: AccountId32 = AccountId32::new([1u8; 32]);
+
+/// A helper struct for initializing and finalizing blocks.
+pub struct BlockBuilder<T>(std::marker::PhantomData<T>);
```

Comment:
Curious why you moved it from pop-drink to here?

Reply:
I prefer having code to initialize the TestExternalities and handle logic of block initialization / finalization in pop-sandbox because there are cases that we need to add more `on_initialize()` code to `BlockBuilder`. You can find the `TODO` in that file about an issue with `pallet-aura` because the original `BlockBuilder` in `pop-drink` does not call the hook of `pallet-aura`.